setup.py

```diff
@@ -216,9 +216,8 @@ def run_tests(self):
     tests_require=[
         'absl-py>=0.10.0',
         'mock>=2.0.0',
```

Comment:
We can remove this now - this is the backported library for python2, right?

Reply:
We could remove "mock>=2.0.0" in a future commit. We still have dependencies on "import mock" (https://github.com/google/openhtf/search?q=%22import+mock%22) instead of "from unittest import mock". Let's make a subsequent commit to remove PY2 support (i.e. remove all six dependencies).